packages/office-ui-fabric-react/src/utilities/positioning/Positioning.Interfaces.ts

```diff
@@ -0,0 +1,54 @@
+import { RectangleEdge } from './Positioning.Enums';
+import { DirectionalHint } from '../../common/DirectionalHint';
+
+export interface IPositionInfo {
```

Comment:
I just went and lowercased all of the "sub" files, and here you are TitleCasing everything.

assume camelCasing files unless they represent a class.

And, if you split a class definition into multiple files (.base, .types, etc) the sub part is camelCased.

positioning.types.ts 

That would be more consistent. Can you just stick your types in a single files?

Reply:
definitely.

Reply:
Done